kube/ims-resolver/ims-resolver-deploy.yml

```diff
@@ -91,10 +97,10 @@ spec:
                 secretKeyRef:
                   name: keycloak-user
                   key: password
-            {{ if or (eq .KUBE_NAMESPACE .BRANCH_ENV) (eq .KUBE_NAMESPACE .UAT_ENV) }}
-            - name: NODE_TLS_REJECT_UNAUTHORIZED
-              value: "0"
-            {{ end }}
+            # {{ if or (eq .KUBE_NAMESPACE .BRANCH_ENV) (eq .KUBE_NAMESPACE .UAT_ENV) }}
+            # - name: NODE_TLS_REJECT_UNAUTHORIZED
+            #   value: "0"
+            # {{ end }}
```

Comment:
Can just remove if we don't need anymore. Could also keep in comment for posterity though

Reply:
thinking we can keep this until we do the next service i.e ASC so we know where to point to

Reply:
Yeah no worries

Reply:
Even if you want to keep it, I would add some context comments https://medium.com/imdoneio/the-imdone-pitch-feedback-welcome-386430accf01

I would just delete it tbh, the history is in git.  We are also already doing it in ASC or you can add documentation.  I'm not a big fan of commenting out code